src/brpc/policy/http_rpc_protocol.cpp

```diff
@@ -1146,13 +1146,13 @@ ParseResult ParseHttpMessage(butil::IOBuf *source, Socket *socket,
                 return MakeParseError(PARSE_ERROR_NOT_ENOUGH_DATA);
             }
             // Send 400 back.
-            butil::IOBuf bad_req;
+            butil::IOBuf bad_resp;
```

Comment:
The variable name should be bad_req, which indicates that the response content is 'Bad Request', rather than a 'bad' response.

Reply:
Not a member.